src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,31 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+
+type Props = {
+  todo: Todo;
+  todoCleaner: () => void;
```

Comment:
The method name 'todoCleaner' doesn't follow the recommended naming conventions for event handler functions in React. Consider renaming it to 'onTodoClear' or something similar.